ui/tests/pages/secrets/backend/list.js

```diff
@@ -8,15 +8,21 @@ export default create({
   createIsPresent: isPresent('[data-test-secret-create]'),
   configure: clickable('[data-test-secret-backend-configure]'),
   configureIsPresent: isPresent('[data-test-secret-backend-configure]'),
-
   tabs: collection('[data-test-tab]'),
   secrets: collection('[data-test-secret-link]', {
     menuToggle: clickable('[data-test-popup-menu-trigger]'),
+    id: text(),
+    click: clickable(),
   }),
   menuItems: collection('.ember-basic-dropdown-content li', {
     testContainer: '#ember-testing',
   }),
-
+  delete: clickable('[data-test-confirm-action-trigger]', {
+    testContainer: '#ember-testing',
+  }),
+  confirmDelete: clickable('[data-test-confirm-button]', {
+    testContainer: '#ember-testing',
```

Comment:
This has to do with ember-basic-dropdown and where the popup gets inserted into the DOM. The test application gets rendered inside of #ember-testing, so that's the default container and the "wormhole" gets rendered along side it. If you don't tell ember-cli-page-object where to look, it defaults to the application div as the test container, so it can't find things that render to the wormhole.

Reply:
Ah gotcha, cool thanks for the info (on all of these ^)